pyvista/_typing.py

```diff
@@ -1,11 +1,35 @@
 """Type aliases for type hints."""
 
-from typing import List, Sequence, Tuple, Union
+from __future__ import (
+    annotations,  # Necessary for autodoc_type_aliases to recognize the type aliases
+)
+
+from typing import TYPE_CHECKING, Dict, List, Sequence, Tuple, Union
 
 import numpy as np
 
+from . import _vtk
+
+if TYPE_CHECKING:
+    from .plotting.colors import Color
+
 Vector = Union[List[float], Tuple[float, float, float], np.ndarray]
 VectorArray = Union[np.ndarray, Sequence[Vector]]
 Number = Union[float, int]
 NumericArray = Union[Sequence[Number], np.ndarray]
-Color = Union[str, Sequence[Number]]
+color_like = Union[
+    Tuple[int, int, int],
+    Tuple[int, int, int, int],
+    Tuple[float, float, float],
+    Tuple[float, float, float, float],
+    Sequence[int],
+    Sequence[float],
+    np.ndarray,
+    Dict[str, Union[int, float, str]],
+    str,
+    "Color",
```

Comment:
Then we can probably use `Color` here
```suggestion
    Color,
```

Reply:
Hmm, when I try that I get `NameError: name 'Color' is not defined` when trying to import pyvista.

Reply:
Oh, you're right. I forgot that the import only works for typing and the definition is there at runtime. Sorry, resolving.